ui/raidboss/emulator/data/NetworkLogConverter.worker.ts

```diff
@@ -1,17 +1,26 @@
 import Encounter from './Encounter';
 import LogEventHandler from './LogEventHandler';
+import LineEvent from './network_log_converter/LineEvent';
 import LogRepository from './network_log_converter/LogRepository';
 import NetworkLogConverter from './NetworkLogConverter';
 
 const ctx: Worker = self as unknown as Worker;
 
 ctx.addEventListener('message', (msg) => {
+  // TODO: should this just throw an exception? Or return an error?
+  if (!(msg.data instanceof ArrayBuffer)) {
+    ctx.postMessage({
+      type: 'done',
+    });
+    return;
+  }
+
```

Comment:
This is one of those should-not-happen situations, so it should probably at least log an UnreachableCode to console.

Maybe it makes more sense to postMessage an error instead, and modify raidemulator.js to handle the error case?

Reply:
I think posting an error would be more change than I want in this PR, since there aren't other examples of handling that.  I'll at least log `UnreachableCode` for now with a TODO.